src/components/BookData.js

```diff
@@ -1,31 +1,26 @@
+// import PropTypes from 'prop-types';
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes. This change is required but very tiny so that I can't stop you from continuing your tasks. but Kindly make sure to fix it before you merge.

Reply:
Thank you! It is done.